script.js

```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
+}
+// POPUP-WINDOW
+const popup = {
+  title: 'Multi Post Stories',
+  techs: ['html', 'Bootstrap', 'Ruby on rails'],
+  imgSrc: './images/popup.png',
+  description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  livelink: 'https://have-samuel.github.io/mobile-ver',
+  sourceLink: 'https://github.com/Have-Samuel/mobile-ver',
+};
+
+// const popupWindow = document.querySelector('.popup1');
+
+function creatPopupCard() {
+  const popupWindow = document.createElement('section');
+  document.body.appendChild(popupWindow);
+  popupWindow.classList = 'popup1';
+  const div = document.createElement('div');
+  div.classList = 'head-img';
+  popupWindow.appendChild(div);
+
+  const popupHeading = document.createElement('h3');
+  popupHeading.textContent = popup.title;
+  div.appendChild(popupHeading);
+
+  const popupClose = document.createElement('img');
+  popupClose.src = './images/Close.png';
+  div.appendChild(popupClose);
+
+  const popupSkills = document.createElement('ul');
+
+  for (let i = 0; i < popup.techs.length; i += 1) {
+    const links = document.createElement('li');
+    links.textContent = popup.techs[i];
+    popupSkills.appendChild(links);
+  }
+
+  popupWindow.appendChild(popupSkills);
+
+  const div1 = document.createElement('div');
+  div1.classList = 'img-head';
+  popupWindow.appendChild(div1);
+
+  const img = document.createElement('img');
+  img.src = './images/popup.png';
+  div1.appendChild(img);
+
+  const paragraph = document.createElement('p');
+  paragraph.textContent = popup.description;
+  div1.appendChild(paragraph);
+
+  const div2 = document.createElement('div');
+  div2.classList = 'button-pop';
+  popupWindow.appendChild(div2);
+
+  const popupButton = document.createElement('a');
+  popupButton.textContent = 'See live';
+  const buttonSpan = document.createElement('span');
+  popupButton.appendChild(buttonSpan);
+  const img1 = document.createElement('img');
+  img1.src = './images/popup1.svg';
+  popupButton.appendChild(img1);
+  div2.appendChild(popupButton);
+
+  const popupButton1 = document.createElement('a');
+  popupButton1.textContent = 'See source';
+  const buttonSpan1 = document.createElement('span');
+  popupButton1.appendChild(buttonSpan1);
+  const img2 = document.createElement('img');
+  img2.src = './images/Github1.png';
+  popupButton1.appendChild(img2);
+  div2.appendChild(popupButton1);
+}
+
+creatPopupCard();
+
+const popupWindow = document.getElementsByClassName('popup1')[0];
+const windowClose = document.querySelectorAll('.see-button');
+
+for (let i = 0; i < windowClose.length; i += 1) {
+  windowClose[i].addEventListener('click', () => {
+    console.log('Helo');
```

Comment:
According to general Javascript best practices requirements:

![image](https://user-images.githubusercontent.com/57421823/156448371-13fdc710-fe81-4c2b-962d-93e0e2aa58ea.png)

Therefore:

- [ ] Please, kindly do not push to the repo any `console.log` calls.